.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,7 @@
+{
+  "image": "mcr.microsoft.com/devcontainers/universal:2",
+  "features": {
+    "ghcr.io/devcontainers/features/desktop-lite:1": {}
+  },
+  "forwardPorts": [6080, 5901]
+}
```

Comment:
This will in theory enable us to run Cypress in Codespaces https://github.com/devcontainers/features/tree/main/src/desktop-lite